src/slider/Slider.tsx

```diff
@@ -180,6 +177,7 @@ const Slider: RneFunctionComponent<SliderProps> = (props) => {
     } else {
       didMountRef.current = true;
     }
+    prevPropValue.current = props.value || 0;
```

Comment:
I am not sure this will make any difference

Reply:
Same! @arpitBhalla